Makefile

```diff
@@ -20,6 +20,7 @@ html-coverage:
 	@sh ./scripts/coverage.sh --html
 
 build:
+	go generate -x ./...
```

Comment:
This should be part of the build and it should check that there are no changes. However, I'm going to do this when I migrate to Github Actions so don't worry about it. Just a FYI

Reply:
ok, thanks